pyproject.toml

```diff
@@ -29,7 +29,7 @@ dependencies = [
     "networkx>=2.2,<3.2",
     "numpy>=1.18.5,!=1.23.0,<1.26",
     "pandas[parquet,excel,fss,gcp,compression]>=2.0,<2.1",
-    "pyarrow>=5,<13.1",
+    "pyarrow>=7,<13",
```

Comment:
Glad the fix was to bump the lower end instead of the bring the higher end down :)

Reply:
No no, sadly it was bringing the top down. We'll have to figure out wtf is going on here, or maybe we'll get lucky and it'll be fixed when we update to Pandas 2.1.  I raised the minimum version since I know that's going to be the minimum allowed version in Pandas when PyArrow becomes a mandatory dependency.